lexico/cli.py

```diff
@@ -75,9 +76,23 @@ def init():
 @lexico.command()
 def view():
     '''Lists all the words present in your dictionary.'''
-    #TODO: Option for file output
     #TODO: More information/table columns
     words = get_words()
     formatted_words = format_words(words)
     display_words = tabulate_words(formatted_words)
     click.echo_via_pager(display_words)
+
+
+@lexico.command()
+def export():
+    '''Exports the current vocabulary into a CSV file.'''
+    words = get_words()
+    formatted_words = format_words_for_export(words)
+    with open('vocabulary.csv', 'w', newline='') as outputFile:
+        outputWriter = csv.writer(outputFile)
+        outputWriter.writerow(['WORD','LOOKUPS','CREATED AT','LAST LOOKUP'])
+        outputWriter.writerow(['','','',''])
+        for word in formatted_words:
+            outputWriter.writerow(list(word))
+        outputFile.close()
```

Comment:
You don't need to close the file explicitly. This is the whole point of using a context manager.